server/model/RedditTest.py

```diff
@@ -0,0 +1,44 @@
+#initial tests
+
+import praw
+import os
+from praw.models import MoreComments
+from dotenv import load_dotenv
+
+
+reddit = praw.Reddit("test") # praw.ini file has credetials
+reddit.read_only = True
+print(reddit.read_only)
+    
```

Comment:
can you turn this into a class that implements the methods git chamber and get chamber members? 
see https://github.com/chumomega/echo_chamber/blob/main/server/model/YoutubeClient.py for example

Reply:
If we're going to have those methods in multiple places...should we consider a base class?

Reply:
yup!